packages/core/__tests__/keyframes.js

```diff
@@ -113,6 +113,74 @@ cases(
         )
       }
     },
+    'object with animationName and opaque keyframes as value': {
+      render: () => {
+        let animation = keyframes(css`
+          from {
+            color: green;
+          }
+          to {
+            color: hotpink;
+          }
+        `)
+
+        return (
+          <div
+            css={{
+              animationName: animation,
+              animationDuration: '1s'
+            }}
+          >
+            {animation.name}
+          </div>
+        )
+      }
+    },
+    'object with animationName and string keyframes as value': {
+      render: () => {
+        let animation = keyframes(css`
+          from {
+            color: green;
+          }
+          to {
+            color: hotpink;
+          }
+        `)
+
+        return (
+          <div
+            css={{
+              animationName: `${animation}`,
+              animationDuration: '1s'
+            }}
+          >
+            {animation.name}
+          </div>
+        )
+      }
+    },
+    'object with animation and opaque keyframes as value': {
+      render: () => {
+        let animation = keyframes(css`
+          from {
+            color: green;
+          }
+          to {
+            color: hotpink;
+          }
+        `)
+
+        return (
+          <div
+            css={{
+              animation
```

Comment:
This won't do anything, you need to at least also specify the duration of the animation. (hence the previous comment)

Reply:
I can add animationDuration here to reflect better real-world scenario, but unfortunately animation can contain just an animation name and it's a valid CSS:
https://codesandbox.io/s/sad-cookies-oj0wk